dataproc_jupyter_plugin/handlers.py

```diff
@@ -21,6 +21,7 @@
 import datetime
 
 from google.cloud.jupyter_config.config import gcp_kernel_gateway_url
+from google.cloud.jupyter_config.config import get_gcloud_config
```

Comment:
Nit, should this (and the previous line) be:

`from google.cloud.jupyter_config.config import gcp_kernel_gateway_url, get_gcloud_config`